include/Functions.hpp

```diff
@@ -273,6 +273,31 @@ RLCPPAPI inline void DrawTextPro(const Font& font, const std::string& text, Vect
     ::DrawTextPro(font, text.c_str(), position, origin, rotation, fontSize, spacing, tint);
 }
 
+/**
+ * Get default font
+ */
+RLCPPAPI inline ::Font GetFontDefault() {
+    return ::GetFontDefault();
+}
+
+/**
+ * Load font from file (filename must include file extension)
+ */
+RLCPPAPI inline ::Font LoadFont(const std::string& fileName) {
+    return ::LoadFont(fileName.c_str());
+}
+
+RLCPPAPI inline ::Font LoadFontFromImage(const Image& image, ::Color key, int firstChar) {
+    return ::LoadFontFromImage(image, key, firstChar);
+}
+
```

Comment:
This looks to be pretty much the same as raylib's. Could likely live without it :wink: 
``` suggestion
```